pkg/kubelet/kubelet.go

```diff
@@ -75,8 +75,19 @@ const (
 	initialNodeStatusUpdateFrequency = 100 * time.Millisecond
 	nodeStatusUpdateFrequencyInc     = 500 * time.Millisecond
 
-	// The retry count for updating node status at each sync period.
-	nodeStatusUpdateRetry = 5
+	// Node status update frequency and retry count.
```

Comment:
Please make this sentence more verbose. "nodeStatusUpdateFrequency specifies how often Kubelet XXX. nodeStatusUpdateRetry specifies how many times Kubelet retries XXX."